helper_scripts/cloudstack/setup_cloudstackOps.sh

```diff
@@ -1,38 +0,0 @@
-#!/bin/bash
```

Comment:
Please keep this file as it is used for CloudStackOps toolbox. Not related to Marvin change.